cmd/integration_link.go

```diff
@@ -356,9 +356,12 @@ List of available integrations:
 	}
 
 	integrationLinkManualDeployCommand = cli.Command{
-		Name:      "integration-link-manual-deploy",
-		Category:  "Integration Link",
-		Flags:     []cli.Flag{&appFlag},
+		Name:     "integration-link-manual-deploy",
+		Category: "Integration Link",
+		Flags: []cli.Flag{
+			&appFlag,
+			&cli.BoolFlag{Name: "follow", Usage: "Follow deployment"},
```

Comment:
let's also allow the short version of the flag
+ use the same wording as the `deployment-follow` command

```suggestion
			&cli.BoolFlag{Name: "follow", Aliases: []string{"f"}, Usage: "Follow deployment event stream"},
```